code/game/objects/items/weapons/trick.dm

```diff
@@ -0,0 +1,675 @@
+			if(9)
+				visible_message("<span class='notice'>The card explodes.</span>")
+				explosion(src.loc,-1, 0, 2, 3, 0)
+			if(10)
+				visible_message("<span class='notice'>The card emits a high frequency vibration.</span>")
+				user.Stun(6)
+				user.Weaken(6)
+				user.stuttering = 6
+				playsound(get_turf(src), 'sound/weapons/taserhit.ogg', 100, 1, -1)		
+			if(11)
+				visible_message("<span class='notice'>The card mysteriously turns into a feral cat.</span>")
+				var/deliveryamt = 1
+				var/spawner_type = /mob/living/simple_animal/hostile/feral_cat
+				if(spawner_type && deliveryamt)
+					var/turf/T = get_turf(src)
+					playsound(T, 'sound/effects/phasein.ogg', 100, 1)
+					for(var/i=1, i<=deliveryamt, i++)
+						var/atom/movable/x = new spawner_type
+						x.loc = T
+			if(12)
+				visible_message("<span class='notice'>The card mysteriously vanishes.</span>")
+				user.adjustBruteLoss(-10)//takes away damage from throw force
+			if(13)
+				var/obj/item/weapon/reagent_containers/food/snacks/cookie/C = new(get_turf(src))
+				visible_message("<span class='notice'>The card mysteriously turns into a cookie.</span>")	
+				C.name = "Cookie of Tricks"
+			if(14)
+				user.eye_blind = 10
+				visible_message("<span class='notice'>The card cuts [user]'s eyes.</span>")
+			if(15)
+				visible_message("<span class='notice'>The card hits [user]'s throat.</span>")
+				user.adjustOxyLoss(-50)
+			if(16)
+				visible_message("<span class='notice'>The card produces a bang!</span>")
+				var/turf/T = get_turf(user)
+				for(var/mob/living/M)
+					M.show_message("<span class='warning'>BANG</span>", 2)
+					playsound(loc, 'sound/effects/bang.ogg', 25, 1)
+			
+					var/ear_safety = M.check_ear_prot()
+					var/distance = max(1,get_dist(src,T))
+
+					if(M.weakeyes)
+						M.visible_message("<span class='disarm'><b>[M]</b> screams and collapses!</span>")
+						M << "<span class='userdanger'><font size=3>AAAAGH!</font></span>"
+						M.Weaken(15) //hella stunned
+						M.Stun(15)
+						M.eye_stat += 8
+
+					if(M.flash_eyes(affect_silicon = 1))
+						M.Stun(max(10/distance, 3))
+						M.Weaken(max(10/distance, 3))
+
+
+					if(!ear_safety)
+						M.Stun(max(10/distance, 3))
+						M.Weaken(max(10/distance, 3))
+						M.setEarDamage(M.ear_damage + rand(0, 5), max(M.ear_deaf,15))
+						if (M.ear_damage >= 15)
+							M << "<span class='warning'>Your ears start to ring badly!</span>"
+							if(prob(M.ear_damage - 10 + 5))
+								M << "<span class='warning'>You can't hear anything!</span>"
+								M.disabilities |= DEAF
+						else
+							if (M.ear_damage >= 5)
+								M << "<span class='warning'>Your ears start to ring!</span>"
+			if(17)
+				user.sleeping = 10
+				visible_message("<span class='notice'>The card seems to cause sleepiness.</span>")
+			if(18)
+				visible_message("<span class='notice'>The card seems to cause hallucination.</span>")	//hallucination sucks so im adding more to stack them up to make them better
```

Comment:
should be "hallucinations" not the singular.
Why does it do this 5 times anyway?